content/static/kitchen-sink/javascripts/kitchen-sink.js

```diff
@@ -547,7 +547,7 @@ $(document)
     // tab
     $('.tab.demo .menu .item')
       .tab({
-        history: true,
+     // history: true,
```

Comment:
Why commenting this line out made it work? Don't quite understand. Could you explain something for me?